examples/kubernetes/connectivity-check/README.md

```diff
@@ -2,3 +2,35 @@
 
 Set of deployments that will perform a series of connectivity checks via
 liveness and readiness checks. An unhealthy/unready pod indicates a problem.
+
+## Connectivity checks
+
+* [Standard connectivity checks](./connectivity-check.yaml)
+* [Standard connectivity checks with hostport](./connectivity-check-hostport.yaml)
+  * Requires either eBPF hostport to be enabled or portmap CNI chaining.
+* [Single-node connectivity checks](./connectivity-check-single-node.yaml)
+  * Standard connectivity checks minus the checks that require multiple nodes.
+* [Proxy connectivity checks](./connectivity-check-proxy.yaml)
+  * Extra checks for various paths involving Layer 7 policy.
+
+## Developer documentation
+
+These checks are written in [CUE](https://cuelang.org/) to define various
+checks in a concise manner. The definitions for the checks are split across
+multiple files per the following logic:
+
+* `resources.cue`: The main definitions for templating all Kubernetes resources,
+  including Deployment, Service and CiliumNetworkPolicy.
+* `echo-servers.cue`: Data definitions for all `echo-*` servers used for other
+  connectivity checks.
+* `defaults.cue`: Default parameters used to define how specific checks connect
+  to particular echo servers, including selecting the probe destination;
```

Comment:
I'm finding this semi-colon confusing. Could it be replace with an "and"?

Reply:
it's poorly placed :-) "x and y and z" is an unusual way to list things though. I think this is another job for the Oxford comma.